### DISCUSSION TITLE
Task type mapfile-writer doesn't exist

### DISCUSSION THREAD

**Discussion by IgorJLR** [mapsforge/mapsforge#1557] [Q&A] [answered]
GRAVE: Execution aborted.
org.openstreetmap.osmosis.core.OsmosisRuntimeException: Task type mapfile-writer doesn't exist.

**Comment by devemux86** [comment]
Check the [map writer documentation](https://github.com/mapsforge/mapsforge/blob/master/docs/Getting-Started-Map-Writer.md#plugin-installation) about how to install the map writer plugin in Osmosis.

**Reply by IgorJLR** [reply]
It doesn't work on Windows systems, I've tried all the solutions in the documentation and forums without success.

On my machine the program is located in "C:\Program Files (x86)\osmosis" with the plugin "mapsforge-map-writer-master-20240603.115659-552-jar-with-dependencies.jar" in the "lib/default" folder ", have already added more memory with the "JAVACMD_OPTIONS" command but the error persists,

**Reply by devemux86** [reply, marked as answer]
Need to read carefully the [Osmosis plugin guide](https://wiki.openstreetmap.org/wiki/Osmosis/Detailed_Usage_0.48#Plugin_Tasks).
(mentioned in map writer documentation)

Osmosis guide contains possible paths to install plugins on Windows.

**Reply by IgorJLR** [reply]
I was able to export to .map on a Linux system by exporting via the official open street map website instead of the JOSM application.
